src/game/TileEngine/TileDef.cc

```diff
@@ -116,7 +117,35 @@ void CreateTileDatabase()
 			}
 
 			SetSpecificDatabaseValues(cnt1, gTileDatabaseSize, TileElement, TileSurf->bRaisedObjectType);
-
+			// fix incorrect double door flags. in vanilla it only affects DOOR3 in PALACE! tileset and DOOR1 in QUEEN'S TROPICAL
```

Comment:
Ideally this whould be in a separate commit.

Reply:
Got it. I'll keep that in mind in the future.